securitas/tests/unit/conftest.py

```diff
@@ -83,6 +83,26 @@ def dummy_user(make_user):
     yield
 
 
+@pytest.fixture
+def dummy_group():
+    ipa_admin.group_add('dummy-group', description="A dummy group")
+    yield
+    ipa_admin.group_del('dummy-group')
+
+
+@pytest.fixture
+def dummy_user_as_group_manager(logged_in_dummy_user, dummy_group):
+    """Make the dummy user a manager of the dummy-group group."""
+    ipa_admin.group_add_member("dummy-group", users="dummy")
+    ipa_admin.group_add_member_manager("dummy-group", users="dummy")
+    yield
+
+
+@pytest.fixture
+def remove_password_min_time():
```

Comment:
Until now the fixtures have been named after the state they create, not as "actions", what would you say about renaming this one to something like "no_password_min_time"? (very very minor nitpick)